oxidation/client/android/app/src/main/AndroidManifest.xml

```diff
@@ -1,6 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="com.scille.parsec">
+<manifest xmlns:android="http://schemas.android.com/apk/res/android">
```

Comment:
What is this for ?

Reply:
see https://github.com/Scille/parsec-cloud/pull/3732#discussion_r1043201769